config/locales/en.yml

```diff
@@ -3119,184 +3147,204 @@ en:
-            <li>Any age and had a permanent disability that prevented you from working and making money</li>
-            </ul></li>
-            <li class="spacing-below-25"><p>3. You lived:</p>
-            <ul><li>with that person in the United States for more than 6 months in 2024.</li>
-            </ul></li>
-            <li><p>4. You aren't filing a joint return with your spouse for the year. Or you're filing a joint return with your spouse only to get a refund of money you paid toward taxes.</p></li>
-            </ol>
-            <p>If <strong>all</strong> of these statements are true, you are the qualifying child of another taxpayer for EITC. This is the case even if they don't claim the credit or don't meet all of the rules to claim it.</p>
-          helper_heading: How do I know if I could be someone else's qualifying child for EITC?
-          instructions: We just have to make sure that you are not someone else's qualifying child for this credit first.
-          primary_eitc_qualifying_child_question: In %{filing_year}, could you be someone else's qualifying child for the Earned Income Tax Credit?
-          spouse_eitc_qualifying_child_question: In %{filing_year}, could your spouse be someone else's qualifying child for the Earned Income Tax Credit?
-          title: You may be eligible for the Earned Income Tax Credit
+          helper_contents_html: "<p>You are the qualifying child of another taxpayer for the New Jersey Earned Income Tax Credit (NJEITC) in %{filing_year} if all of these are true.</p>\n<ol class=\"list--numbered\">\n  <li>You are that person's:\n    <ul class=\"list--bulleted\">\n      <li>Child, stepchild, foster child, or a descendant of any of them, or</li>\n      <li>Sibling, half sibling, step sibling, or a descendant of any of them</li>\n    </ul>\n  </li>\n  <li>You were: \n    <ul class=\"list--bulleted\">\n      <li>Under age 19 at the end of the year and younger than that person (or that person's spouse, if they filed jointly), or</li>\n      <li>Under age 24 at the end of the year, a student, and younger than that person (or that person’s spouse, if they filed jointly), or</li>\n      <li>Any age and had a permanent disability that prevented you from working and making money</li>\n    </ul>\n  </li>\n  <li>You lived with that person in the United States for more than 6 months in %{filing_year}.</li>\n  <li>You aren’t filing a joint return with your spouse for the year. Or you’re filing a joint return with your spouse only to get a refund of money you paid toward taxes.</li>\n</ol>\n<p>If all of these statements are true, you are the qualifying child of another taxpayer for NJEITC. This is the case even if they don’t claim the credit or don’t meet all of the rules to claim it.</p>\n<p> If only some of these statements are true, you are NOT the qualifying child of another taxpayer for NJEITC.</p>"
```

Comment:
[dust] This is harder to read than before. Why use quotes and not the | ?

Reply:
The `i18n-normalize` seems to randomly combine some (but not all) of my multi-line `|`. I was using the | for all of them when editing... not sure how to format it to prevent it from mushing my multilines together...